example/src/index.js

```diff
@@ -1,10 +1,10 @@
 import FlameChart from './../../dist/index.js';
-import { defaultTimeGridSettings } from '../../dist/engines/time-grid.js';
-import { defaultRenderSettings } from '../../dist/engines/basic-render-engine.js';
-import { defaultTimeGridPluginSettings } from '../../dist/plugins/time-grid-plugin';
-import { defaultTimeframeSelectorPluginSettings } from '../../dist/plugins/timeframe-selector-plugin';
-import { defaultTogglePluginSettings } from '../../dist/plugins/toggle-plugin.js';
-import { defaultWaterfallPluginSettings} from '../../dist/plugins/waterfall-plugin.js';
+import { defaultTimeGridStyles } from '../../dist/engines/time-grid.js';
+import { defaultRenderStyles } from '../../dist/engines/basic-render-engine.js';
+import { defaultTimeGridPluginStyles } from '../../dist/plugins/time-grid-plugin';
+import { defaultTimeframeSelectorPluginStyles } from '../../dist/plugins/timeframe-selector-plugin';
+import { defaultTogglePluginStyles } from '../../dist/plugins/toggle-plugin.js';
+import { defaultWaterfallPluginStyles} from '../../dist/plugins/waterfall-plugin.js';
```

Comment:
I think it would be good to have add an eslint rule to have consistent curlies style
now it is mixed with `{...}`, `{ ...}` or `{ ... }`

Reply:
I can have a look into that when I work on some other PR